.github/workflows/graphene_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   test:
     strategy:
       matrix:
-        llvm-version: [15, 16]
+        llvm-version: [16, 17]
```

Comment:
Yeah, no reason to support LLVM 15 anymore, Manjaro's on 16.

Reply:
Mhm, I think last two is pretty reasonable